src/PerfView/PerfViewData.cs

```diff
@@ -9700,6 +9712,7 @@ protected override Action<Action> OpenImpl(Window parentWindow, StatusBar worker
                 m_Children.Add(advanced);
             }
 
+            m_hasUniversal = hasUniversalSystem || hasUniversalCPU || hasUniversalCSwitch;
```

Comment:
[cautious Mark] This looks correct and reasonable. However, it highlights a peril that any future change that adds an "early return" will miss setting this and this method is large enough and this setting is removed enough from the has* sets above that this could easily happen. Consider setting m_hasUniversal to true at the same time as in individual items.

Also, why the heck does this always return null, lol? I thought for sure I was going to find an earlier return when reviewing this.